### PR TITLE
[AB#39028] Upgrade to Node.js 18

### PR DESCRIPTION
**Pull request by BenjaminSchwendner**
# Pull Request

## Prerequisites

- [x] My code follows the coding conventions
- [x] All tests pass

## Description

This PR upgrade the media-template to Node.js version 18 (current LTS version).
It also bumps the Piral version and starts making use of [import-maps](https://docs.piral.io/concepts/I08-importmap).
The jest configurations are also migrated to use TypeScript instead of JavaScript.

### CODE REVIEW

File: services/catalog/service/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   ],
   "main": "dist/index.js",
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
   },
   "scripts": {
     "clean": "rimraf dist",
```
Comment (binaryunary) [inline services/catalog/service/package.json:18]: test

File: .adops/docker/Dockerfile.backend
Comment (EmperorRXF) [inline .adops/docker/Dockerfile.backend:1]: potentially improvement task:

@yohanAnushka: we seem to have two `Dockerfiles` in media-template? if they're the same, could we drop the one inside .adops folder?

File: services/media/workflows/pilet.json
Comment (EmperorRXF) [inline services/media/workflows/pilet.json:1]: who is the consumer of this file? guess it's used by piral-cli? but wanted to understand what usecase this file servers

**Review by EmperorRXF — APPROVED** [review]

File: .adops/docker/Dockerfile.backend
Comment (yohanAnushka) [reply .adops/docker/Dockerfile.backend:1]: Refactored to use the dockerfile in the root of the repo.

File: services/media/workflows/pilet.json
Comment (BenjaminSchwendner) [reply services/media/workflows/pilet.json:1]: yes, this is the piral-cli's "rc-file".
Previously the information was located inside the `package.json`.
It's mainly a mapper to make the CLI aware of which piralInstance to run the Pilet in, but also allows for e.g. defining the packaging format version of the pilet.

**Review by EmperorRXF — APPROVED** [review]

**Pull request merged**